### ISSUE TITLE
Question about Sigma

### ISSUE THREAD

**Issue by xinwei-git** [zifuwan/Sigma#51]
May I ask, for this training, have the training set and validation set been merged? That is, is train2 used for training?
